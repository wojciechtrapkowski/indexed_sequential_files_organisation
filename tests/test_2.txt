insert 20 3 
insert 10 2
print
search 20
search 10
