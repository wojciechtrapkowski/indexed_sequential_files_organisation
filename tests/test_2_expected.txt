

================================================
Index area
================================================
Page 0 number of entries: 1
	Entry 0
		start_key: 20
		page_index: 1
================================================
Main area
================================================
Guardian overflow page index: 0

Page 0 number of entries: 1
	Entry 0
		key: 20
		value: 3
		overflow_entry_index: null
================================================
Overflow area
================================================
Page 0 number of entries: 1
	Entry 0
		key: 10
		value: 2
	 	overflow_entry_index: null
3
2
